src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -169,11 +153,30 @@ class RelaxedRigidContacts(ContactModel):
         default_factory=FlatTerrain.build
     )
 
+    _solver_options_keys: jax_dataclasses.Static[tuple[str, ...]] = dataclasses.field(
+        default=("tol", "maxiter", "memory_size"), kw_only=True
+    )
+    _solver_options_values: jax_dataclasses.Static[tuple[Any, ...]] = dataclasses.field(
+        default=(1e-6, 50, 10), kw_only=True
+    )
+
+    @property
+    def solver_options(self) -> dict[str, Any]:
+
+        return dict(
+            zip(
+                self._solver_options_keys,
+                self._solver_options_values,
+                strict=True,
+            )
+        )
+
```

Comment:
What about moving this to the base class?

Reply:
This is not part of all implementations, `SoftContacts` does not need it.